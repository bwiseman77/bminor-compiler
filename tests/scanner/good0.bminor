return a &&b  ? true : false
+125637
-12837

