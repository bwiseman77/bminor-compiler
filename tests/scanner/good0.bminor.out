-------------------------------------------
           token type  text
-------------------------------------------
              RETURN  return
               IDENT  a
                 AND  &&
               IDENT  b
        TERNARY_PT_1  ?
                TRUE  true
               COLON  :
               FALSE  false
     INTEGER_LITERAL  +125637
     INTEGER_LITERAL  -12837
